mmv1/third_party/terraform/.teamcity/components/inputs/services_ga.kt

```diff
@@ -436,6 +436,11 @@ var ServicesListGa = mapOf(
         "displayName" to "Iam2",
         "path" to "./google/services/iam2"
     ),
+    "iam3" to mapOf(
+        "name" to "iam3",
+        "displayName" to "Iam3",
+        "path" to "./google/services/iam3"
+    ),
```

Comment:
This is a consequence of making the new resource (and the wider service) beta-only.

```suggestion
```